src/components/custom/_signup-public.tsx

```diff
@@ -14,6 +14,7 @@ import Apple from 'images/svg/custom/apple-40.svg'
 import Facebook from 'images/svg/custom/facebook-40.svg'
 import Google from 'images/svg/custom/google-40.svg'
 import Arrow from 'images/svg/custom/chevron-right.svg'
+import { Desktop, Mobile } from 'components/containers/visibility'
```

Comment:
This can be imported directly from `components/containers`

Reply:
ok